components/Themes/src/MaterialThemes.h

```diff
@@ -0,0 +1,18 @@
+/*
+ Copyright 2015-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCTheme.h"
+#import "MDCFeatureHighlightColorThemer.h"
```

Comment:
I believe we have two options:
Themers live next to their components and every component depends on MaterialThemes.
Themers live in MaterialThemes and depend on every component.